dashboard/db/schema_cache.yml

```diff
@@ -7130,7 +7130,7 @@ columns:
     comment: 
   lessons_opportunity_standards:
   - &1825 !ruby/object:ActiveRecord::ConnectionAdapters::MySQL::Column
-    name: lesson_id
+    name: id
```

Comment:
I think the changes in this file are not relevant to your change, so let's remove any changes to this file from this PR.

Reply:
Confirmed as irrelevant. Removed!